deployment/Deploy.ps1

```diff
@@ -241,19 +214,24 @@ process {
         }
     
         #Creating EdnaLiteDevKey in keyVault and Updating the Config Entry EdnaLiteDevKey in the Function Config
-        $ke = az keyvault key create --vault-name $deploymentOutput.properties.outputs.KeyVaultName.value --name EdnaLiteDevKey --protection software
+        $keyCreationOp = az keyvault key create --vault-name $deploymentOutput.properties.outputs.KeyVaultName.value --name EdnaLiteDevKey --protection software
+        if(!$keyCreationOp) {
+            throw "Encountered an Error while creating Key in keyVault"
+        }
         $KeyVaultLink = $(az keyvault key show --vault-name $deploymentOutput.properties.outputs.KeyVaultName.value --name EdnaLiteDevKey --query 'key.kid' -o json);
```

Comment:
Can we get rid of this command, and use the output of above command itself?

Reply:
Also, since this something that can be done manually as well, would it better to also direct the user on how to do this manually 🤔?

Reply:
this can be done manually but then the user will also have to set the key in the required functions apps' configs

Reply:
>Can we get rid of this command, and use the output of above command itself?

I'll see if this can be done somehow